src/framework/preferences/css/stylus/SeparatedPanelPrefsEditorFrame.styl

```diff
@@ -151,10 +151,11 @@ html {
                 .fl-prefsEditor-panel-descr {
                     clip: auto;
```

Comment:
I think we actually want the content to be visible though. So i figure it's better to ensure that it is or at least be explicit.